docs/source/creators/creators_description_classes.rst

```diff
@@ -34,6 +34,9 @@ For each output class, a separate vector layer can be created.
 
 Output report contains information about percentage coverage of each class.
 
+The model should have at least two output classes, one for the background and one (or more) for the object of interest. The background class should be the first class in the output.
```

Comment:
The model can also have one output (1, h, w) or even no channel (h, w) when binary segmentation with sigmoid (instead of softmax) is performed. We should handle each of these cases.

Reply:
TODO update